source/Calamari.Terraform.Tests/CommandsFixture.cs

```diff
@@ -172,7 +171,7 @@ async Task DownloadCli(string destination, string version)
 
             if (Directory.Exists(destinationDirectoryName))
             {
-                var path = Directory.EnumerateFiles(destinationDirectoryName).FirstOrDefault();
+                var path = Directory.EnumerateFiles(destinationDirectoryName).FirstOrDefault(f => Path.GetFileName(f).Contains("terraform"));
```

Comment:
They started adding `LICENSE.txt` to the download, which is the first file, which is not the executable

Reply:
![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExdWFmdDEwcnd5NHFnMDM4ajJtNGx6dGVnemhybmI4YWNvbGJ1d3oyayZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/ac7MA7r5IMYda/giphy.gif)